src/parsec.rs

```diff
@@ -194,7 +197,30 @@ fn create(full_id: FullId, gen_pfx_info: &GenesisPfxInfo) -> Parsec {
             &gen_pfx_info.first_info.members(),
             &gen_pfx_info.latest_info.members(),
             ConsensusMode::Single,
-            Box::new(rand::os::OsRng::new().unwrap()),
+            rng,
         )
     }
 }
+
+// Create new Rng instance.
+//
+// In production, use `OsRng` for maximum cryptographic security.
+//
+// In test, use a weaker rng to prevent creating too many open file handles (as each `OsRng`
+// instance might internally contain a handle to /dev/random or similar). This avoid a bug with too
+// many open files which sometimes happens because some test create lot of nodes and each node has its own rng.
```

Comment:
what's your setting for the number of chars each line?
I think we have the perference of 100 and seems the last line exceeds it?